### ISSUE TITLE
Issues about quantize the vgg-SSD model

### ISSUE THREAD

**Issue by wangxianrui** [Tencent/PocketFlow#194]
**Describe the bug**
In the full-learner, we train vgg-ssd and add 'images-final' and 'logits-final' to tensorflow collections, but the images-filanl and logits-final we want to add are all dict, and can't be added to. So this problem makes the later operations ( such as convert .ckpt to .pb and .tflie) failed.  

`WARNING:tensorflow:Issue encountered when serializing images_final.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'dict' object has no attribute 'name'
WARNING:tensorflow:Issue encountered when serializing logits_final.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'dict' object has no attribute 'name'
`

**Comment by jiaxiang-wu** [comment]
Thanks for the reported bug. We have also noticed this issue, and a fix will be added along with the uniform quantization support for SSD models.

**Comment by jiaxiang-wu** [comment]
In PR #199, we have fixed this issue:
https://github.com/Tencent/PocketFlow/pull/199/files#diff-d3cde232530ead961d9366a9ea7d050fR308
